apps/web-giddh/src/app/services/receipt.service.ts

```diff
@@ -289,4 +308,44 @@ export class ReceiptService implements OnInit {
             .replace(':purchaseRecordUniqueNumber', purchaseRecordUniqueName)
         ).pipe(catchError((e) => this.errorHandler.HandleCatch<Voucher, any>(e)));
     }
+
+    /**
+     * Fetches all the advance receipts
+     *
+     * @param {GetAllAdvanceReceiptsRequest} requestObject Request object
+     * @returns {Observable<BaseResponse<any, GetAllAdvanceReceiptsRequest>>} Observable to carry out further operations
+     * @memberof ReceiptService
+     */
+    public getAllAdvanceReceipts(requestObject: GetAllAdvanceReceiptsRequest): Observable<BaseResponse<any, GetAllAdvanceReceiptsRequest>> {
+        const companyUniqueName = this._generalService.companyUniqueName;
```

Comment:
Don't use companyUniqueName from general service. It will come wrong if we will switch company.

Reply:
Done, Have change in header as header will call every company change